scripts/install-deps.sh

```diff
@@ -156,6 +161,25 @@ do_install_githooks() {
     fi
 }
 
+do_install_openssl() {
+    local openssl_dir=$work_folder/openssl-1.1.1f
+    echo "Installing OpenSSL ..."
+
+    $SUDO apt-get install -y build-essential checkinstall zlib1g-dev -y || return
+    wget https://www.openssl.org/source/openssl-1.1.1f.tar.gz || return
+    mkdir -p $openssl_dir || return
+    tar -xf openssl-1.1.1f.tar.gz -C $openssl_dir --strip-components=1 || return
```

Comment:
move `openssl-1.1.1f` to variable at top of file (should be `openssl-1.1.1u` I think) so can easily update the openssl version when new vulnerabilities are fixed in `u` and need to move to a future `v`